x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_forms/config_form.tsx

```diff
@@ -25,14 +25,27 @@ const PolicyDetailCard = styled.div`
   }
 `;
 export const ConfigForm: React.FC<{
+  /**
+   * A subtitle for this component.
+   **/
   type: string;
-  supportedOss: string[];
+  /**
+   * Types of supported operating systems.
+   */
+  supportedOss: React.ReactNode;
   children: React.ReactNode;
-  id: string;
-  /** Takes a react component to be put on the right corner of the card */
+  /**
+   * A description for the component.
+   */
+  description: string;
+  /**
+   * The `data-test-subj` attribute to append to a certain child element.
+   */
+  dataTestSubj: string;
```

Comment:
renaming from `id` because its just used for `data-test-subj` and `id` already has other meanings